manifests/golang.yml

```diff
@@ -355,9 +355,9 @@ tests/:
       Test_TelemetryInstallSignature: missing_feature
     test_trace_sampling.py:
       Test_Trace_Sampling_Basic: v1.37.0 # TODO what is the earliest version?
-      Test_Trace_Sampling_Globs: missing_feature
-      Test_Trace_Sampling_Resource: missing_feature
-      Test_Trace_Sampling_Tags: missing_feature
+      Test_Trace_Sampling_Globs: v1.60.0-dev
+      Test_Trace_Sampling_Resource: v1.60.0-dev
+      Test_Trace_Sampling_Tags: v1.60.0-dev
     test_tracer.py:
```

Comment:
The support for "Test_Trace_Sampling_With_W3C" is since....

Reply:
1.60.0 is good

Reply:
Same, 1.60.0 is fine, changed.